docs/content/users/quickstart.md

```diff
@@ -4,12 +4,12 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
 
 === "Craft CMS"
 
-    ## Craft CMS
+    ## Craft CMS (4)
 
     Start a new [Craft CMS](https://craftcms.com) project or retrofit an existing one.
 
-    !!!tip "Compatibility"
-        The `craft` project type was added to DDEV in version [1.21.2](https://github.com/ddev/ddev/releases/tag/v1.21.2). Check your current version with the `ddev version` command, and [upgrade](../users/usage/faq.md#how-can-i-updateupgrade-ddev) if necessary!
+    !!!tip "Compatibility with Craft CMS 3"
+        The `craftcms` project type is best with Craft CMS 4+. If you are using Craft CMS 3 you may prefer to use the `php` project type. Craft CMS 4+ (and thus DDEV's support) are more opinionated about some settings, and you may want to just manage your own settings. More information is available in [issue](https://github.com/ddev/ddev/issues/4650).
```

Comment:
Can whittle this down for efficiency, grammar, and link context. If I’m following correctly, it’d also be good to mention Craft 4, then Craft 3 and the recommended action. As written the tip mentions Craft 3, then the text mentions Craft 4, Craft 3, and Craft 4 again and it’s a bit of a jumble. I hope this is more straightforward and still what you intended.

```suggestion
        The `craftcms` project type is best with Craft CMS 4+, which is more opinionated about some settings. If you are using Craft CMS 3, you may want to use the `php` project type and [manage settings yourself](https://github.com/ddev/ddev/issues/4650).
```